src/Main/Tasks.cpp

```diff
@@ -69,10 +77,21 @@ static std::map<FOEDAG::Compiler::SynthesisOpt, const char*> synthOptMap = {
     {FOEDAG::Compiler::SynthesisOpt::Delay, "delay"},
     {FOEDAG::Compiler::SynthesisOpt::Mixed, "mixed"},
     {FOEDAG::Compiler::SynthesisOpt::Clean, "clean"}};
+
+static std::map<FOEDAG::Compiler::PlacementOpt, const char*> placeOptMap = {
+    {FOEDAG::Compiler::PlacementOpt::None, "none"},
+    {FOEDAG::Compiler::PlacementOpt::Clean, "clean"},
+    {FOEDAG::Compiler::PlacementOpt::Random, "Random"},
+    {FOEDAG::Compiler::PlacementOpt::InDefineOrder, "In Define Order"}};
```

Comment:
I don't think it should have spaces.

Reply:
@KochynVolodymyr at  85 line "In Define Order"?